libraries/networking/src/webrtc/WebRTCSignalingServer.cpp

```diff
@@ -10,19 +10,70 @@
 
 #if defined(WEBRTC_DATA_CHANNELS)
 
+#include <QSslKey>
 #include <QtCore>
 #include <QWebSocket>
 
+#include <BuildInfo.h>
+#include <PathUtils.h>
+
 #include "../NetworkLogging.h"
 #include "../NodeType.h"
 
 
 const int WEBRTC_SOCKET_CHECK_INTERVAL_IN_MS = 30000;
 
-WebRTCSignalingServer::WebRTCSignalingServer(QObject* parent) :
-    QObject(parent),
-    _webSocketServer(new QWebSocketServer(QStringLiteral("WebRTC Signaling Server"), QWebSocketServer::NonSecureMode, this))
+WebRTCSignalingServer::WebRTCSignalingServer(QObject* parent, bool isWSSEnabled) :
+    QObject(parent)
 {
+    if (isWSSEnabled) {
+        _webSocketServer = (new QWebSocketServer(QStringLiteral("WebRTC Signaling Server"), QWebSocketServer::SecureMode,
+            this));
+
+        auto dsDirPath = PathUtils::getAppLocalDataPath();
+        const QString KEY_FILENAME = "cert.key";
```

Comment:
Same comment re constant locations.

Regarding the file name... 
The cert isn't necessarily just for WebRTC: it could also be used to secure the server settings page as HTTPS.
The cert is for the DNS domain name being used as the address to connect to the domain server at sop could, in theory, be used for other things also. This might change if we switch to using IP address certs or some other SSL method. Perhaps `vircadia-cert.key` might be suitable.

Reply:
My thought re: the cert name: I'd like something unique because I found myself trying to figure out where's a given cert for a given program a few times, and there are cases where you may end up with multiple certs in the same directory. It's just nice to clearly know what a cert is for at a glance.

But yeah, vircadia-cert.key works for me.

Reply:
I just realized one issue... It's entirely possible that multiple domains on the same server may use a different cert. We should probably keep that in mind now as that will most certainly become a thing.